Sprint_Tests/maria_ptitsyna.md

```diff
@@ -0,0 +1,12 @@
+1. He used to manage well in layout and work with actual technologies in front end. After small break in several years and major depression plus listening hard music, he weakly manages in actual tendentious and drinks a lot. If he had overcome the depression after dead of his wife and child, he wouldn't have had a question about search of a job with a relevant salary.
+
+2. If he had a goal and comparable wage expectation, he would already work remotely somewhere in the UAS or in Bali with a perspective to relocate in The Canary Islands and make spanish citizens. Since childhood, he has been dreaming about marriage with spanish girl and got retired early and growing up grape on a seaside of The Balearic Sea.
+
```

Comment:
and adequate/ reasonable salary expectation
wage is paid on a weekly basis
the perspective to relocate (we define what kind of perspective we mean)
to the Canary Islands (preposition of destination)
apply for Spanish citizenship. Language and nationality are always written with the capital letter
Spanish girl
dreaming about marriage ... and early retirement